cms/grading/ScoreType.py

```diff
@@ -73,6 +73,30 @@ def __init__(self, parameters, public_testcases):
         self.max_score, self.max_public_score, self.ranking_headers = \
             self.max_scores()
 
+    def format_score(self, score, max_score, unused_score_details,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/s-maggiolo/cms/file/2782145405/issues/source?bid=3167167&fileBranchId=3365636#l76)